src/ReactiveSocket.cpp

```diff
@@ -75,6 +76,20 @@ void ReactiveSocket::requestSubscription(
   automaton->start();
 }
 
+void ReactiveSocket::requestFireAndForget(
+    Payload request,
+    Subscriber<Payload>& responseSink) {
```

Comment:
Is there a C++ convention for a non instantiable type?  something to indicate that onNext can never be called? Scala would be Nothing.

Or is the benefit of using Payload that we can use the same subscriber classes?